m4_sessions/1710-inning/group_six.md

```diff
@@ -1,11 +1,22 @@
 Group Six
 
+Session Title: "Learning another language"
 
-Session Title: 
-Description: Why should students come to this session? 
+Session Title:Description: Why should students come to this session?
 
+Students will need to be comfortable learning another language by the time they leave Turing if they want to market themselves as more than just Rails- or JavaScript- developers.
 Session Outcomes: What will students leave with or gain from this session?
 
-Session Outline:  Include the flow of the session and the time for each section.
+Students will be better able to start thinking about other languages they may want to learn after Turing (or for their capstones). They will leave with some resources for tutorials, etc. if they want to delve more into a particular language.
+Session Outline: Include the flow of the session and the time for each section.
 
-Names of group members 
+Preface with the fact that many of us will not be working in one language for all of our careers. The ability to learn another language is paramount to demonstrating marketable skills. -Overview of learning any new language
```

Comment:
Really drive this home.  Maybe there is some data on the average number of languages that devs know?  Or maybe pull some job postings that you are each applying for and show what you might need to learn.